### ISSUE TITLE
Latest build broke markdown preview for Microsoft Edge articles

### ISSUE THREAD

**Issue by dan-wesley** [microsoft/vscode#91170]
- VSCode Version: 1.42.1
- OS Version: Windows 10

I was using the Dark (Visual Studio) theme. Basic black background with white text. The doc preview page was white background with black text. The colors for alerts such as Note or Important were what you see when the article goes live on docs.microsoft.com. After the automatic update the preview is now black with white text. The alerts are completely off color from what gets published. Need a workaround or how to go back a build. (Tried to use another preview extension (Github Styling) but the Doc Authoring Pack preview overrides it.)

Does this issue occur when all extensions are disabled?: Yes/No
Can't test, need extensions enabled for markdown format and preview

**Comment by mjbvz** [comment]
So you aren't using VS Code's builtin markdown preview, but one from an extension?

**Comment by dan-wesley** [comment]
Hi Matt,
My basis setup is running the Docs Authoring Pack 0.6.1 with all the items that are in the pack.
Last summer when I started I alternated between the Dark and Dark+ themes. I found that when working with articles that had Alerts and tables it was easier to see how things were going to look if I had the same color scheme as the final doc when published. So last Sept I added the Github Markdown Preview. It was excellent and up until yesterday morning everything worked fine. I tried re-enabling it, but the Github preview flashes on the screen and is replaced by the Authoring Pack preview.
I would really like to get back to where I was –
Thanks
dan


/**
* Dan Wesley (Experis)
*/
